forc-plugins/forc-doc/src/doc.rs

```diff
@@ -95,7 +67,12 @@ impl Document {
                 let desc = Descriptor::from_typed_decl(
                     decl_engine,
                     decl,
-                    ModuleInfo::from_vec(vec![project_name.to_owned()]),
+                    ModuleInfo::from_ty_module(
+                        vec![project_name.to_owned()],
+                        None,
+                        // later we should allow a description in the forc.toml
+                        // to be used as the attributes of the project root
```

Comment:
I don't think we do want this do we? The root module docs are often much more detailed than the `description` that appears in a manifest, I think it's fine to treat these separately.

A more commonly used feature in Rust is the ability to use a README as the root-level docs, e.g.
```rust
#![doc = include_str!("../README.md")]
```
But I don't think `forc doc` itself shouldn't have to worry about this.

Reply:
I'll go ahead and remove the comment, I think maybe I can find a way to make it work with the `lib.sw`, `main.sw` files, or as you said above.

Reply:
Added support for this, it looks like: 

![Screenshot from 2023-02-21 16-01-51](https://user-images.githubusercontent.com/57543709/220468638-ca47c94e-8b20-45a1-88ca-cee0174b34ae.png)